src/foam/nanos/auth/resetPassword/ResetPassword.js

```diff
@@ -10,6 +10,15 @@ foam.CLASS({
 
   documentation: 'Reset Password Base Model',
 
+  imports: [
+    'ctrl',
```

Comment:
Please don't import 'ctrl'. If you need something from ctrl then import that directly, like 'window' or 'stack'.